frontend/views/components/Navbar.js

```diff
@@ -17,31 +17,45 @@ const Navbar = {
                     <div class="collapse navbar-collapse" id="navbarNav">
                     <ul class="navbar-nav">
                         <li class="nav-item">
-                        <a class="nav-link active" href="#/">Home</a>
+                        <a class="nav-link active" href="#/" data-i18n="home">Home</a>
                         </li>
                         <li class="nav-item">
-                        <a class="nav-link active" href="#/gameplay">Gameplay</a>
+                        <a class="nav-link active" href="#/gameplay" data-i18n="gameplay">Gameplay</a>
                         </li>
                         <li class="nav-item">
-                        <a class="nav-link" href="#/tournament">Tournament</a>
+                        <a class="nav-link" href="#/tournament" data-i18n="tournament">Tournament</a>
                         </li>
                         <li class="nav-item">
-                        <a class="nav-link" href="#/setup-otp">Setup Otp</a>
+                        <a class="nav-link" href="#/setup-otp" data-i18n="setup-otp">Setup Otp</a>
                         </li>
                         <li class="nav-item">
-                        <a class="nav-link" href="#/mypage">My Page</a>
+                        <a class="nav-link" href="#/mypage" data-i18n="my-page">My Page</a>
                         </li>
                     </ul>
                     <ul class="navbar-nav ms-auto">
                         <li class="nav-item">
-                        <a class="nav-link" href="#/login">Login</a>
+                        <a class="nav-link" href="#/login" data-i18n="login">Login</a>
                         </li>
                     </ul>
+                    <ul class="navbar-nav ms-auto">
+                      <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="languageDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false" data-i18n="language">
+                          Language
+                        </a>
+                        <ul class="dropdown-menu" aria-labelledby="languageDropdown">
+                          <li><a class="dropdown-item" href="#" onclick="setLanguage('en')" data-i18n="english">English</a></li>
+                          <li><a class="dropdown-item" href="#" onclick="setLanguage('ja')" data-i18n="japanese">Japanese</a></li>
+                        </ul>
+                      </li>
+                    </ul>

```

Comment:
```suggestion
                    <ul class="navbar-nav ms-auto">
                        <li class="nav-item">
                            <a class="nav-link" href="#/login" data-i18n="login">Login</a>
                        </li>
                        <li class="nav-item dropdown">
                          <a class="nav-link dropdown-toggle" href="#" id="languageDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false" data-i18n="language">
                            Language
                          </a>
                          <ul class="dropdown-menu" aria-labelledby="languageDropdown">
                            <li><a class="dropdown-item" href="#" onclick="setLanguage('en')" data-i18n="english">English</a></li>
                            <li><a class="dropdown-item" href="#" onclick="setLanguage('ja')" data-i18n="japanese">Japanese</a></li>
                          </ul>
                        </li>
                    </ul>
```
Loginボタンが微妙な位置にいるので修正して欲しいです(そもそもまとまりになっているLoginボタンと言語選択ボタンで別々のulを使う必要がない)

Reply:
修正しました！